arbitrum-docs/partials/_migrate-nft-collections-partial.md

```diff
@@ -0,0 +1,90 @@
+This how-to is for **users** and **developers** who want to migrate (or "bridge") existing NFT collections onto Arbitrum.
+
+## Why migrate NFTs to Arbitrum?
```

Comment:
The concern I have with this section is that this how-to isn't only targeted at nft collections migrating from ethereum. They could be coming from polygon or solana in which case cheaper/faster txs might not be true

Reply:
Understood - two options come to mind:

1. **Reduce scope of doc**: Aim this only at the `Ethereum` -> `Arbitrum` migration task, and then author separate guidance for those other migrations (ideal if there's not a ton of overlap in the procedure / tradeoffs)

2. **Add precision to value prop**: update the value prop to be true for both `Ethereum` -> `Arbitrum` and `Not-Ethereum` -> `Arbitrum` (ideal if there is a ton of overlap - just pushed a demo of this)

Reply:
There is a ton of overlap with the procedure, so I would advocate for the 2nd option. I do think one of the bigger value props is taking advantage of the Arbitrum ecosystem, users, and liquidity. Many of the projects migrating from other chains like polygon or solana are doing so because they were unsuccessful there and so this is kind of like a fresh start. So maybe interoperability with the "EVM" and becoming part of the Arbitrum Ecosystem are two separate value props? There's also a question if we need to delineate the differences between One and Nova

Reply:
Just pushed an update that sets these expectations explicitly :)